localstack/utils/aws/message_forwarding.py

```diff
@@ -141,6 +154,65 @@ def send_event_to_target(
         LOG.warning('Unsupported Events rule target ARN: "%s"', target_arn)
 
 
+def auth_keys_from_connection(connection: dict):
```

Comment:
nit: We usually use the type hints from the `typing` module - `typing.Dict` in this case.
```
def auth_keys_from_connection(connection: Dict):
    ...
```